components/style/properties/properties.mako.rs

```diff
@@ -561,21 +560,20 @@ pub fn parse_property_declaration_list(context: &ParserContext, input: &mut Pars
         }
     }
     PropertyDeclarationBlock {
-        important: Arc::new(deduplicate_property_declarations(important_declarations)),
-        normal: Arc::new(deduplicate_property_declarations(normal_declarations)),
+        declarations: Arc::new(deduplicate_property_declarations(declarations)),
     }
 }
 
 
 /// Only keep the last declaration for any given property.
-/// The input is in source order, output in reverse source order.
-fn deduplicate_property_declarations(declarations: Vec<PropertyDeclaration>)
-                                     -> Vec<PropertyDeclaration> {
+/// The input and output are in source order
+fn deduplicate_property_declarations(declarations: Vec<(PropertyDeclaration, Importance)>)
+                                     -> Vec<(PropertyDeclaration, Importance)> {
     let mut deduplicated = vec![];
     let mut seen = PropertyBitField::new();
     let mut seen_custom = Vec::new();
     for declaration in declarations.into_iter().rev() {
```

Comment:
Oh, that’s a good point. I’ll change this to eliminate a normal declaration if there’s an important one of the same name (regardless of order).